docs/BRANCHES.md

```diff
@@ -0,0 +1,16 @@
+# Branches used for Storytelling Tool deployment
+
+Various branches are present in [the main fork of this repository](https://github.com/fluid-project/sjrk-story-telling),
+each of which has a specific purpose, such as being used to deploy particular versions of the site. The `master` branch
+is an exception in that it is used both as the main branch of the project from which all new development should be started
+(see [CONTRIBUTING.md](../CONTRIBUTING.md) for more info on that process), as well as the branch from which the Staging
+site is deployed.
+
+Authoring new stories is normally disabled on all deployed versions except the Staging site.
+
+| Branch name                         | Site | Description |
```

Comment:
There's another branch `learning-to-learn` that is not listed here. Can you add it in? Thanks.

Reply:
I was thinking we should remove this branch and left a comment to that effect on [the Jira page](https://issues.fluidproject.org/browse/SJRK-321), so I didn't mention it, `stories-floe-dev` or `re-review`.